client/lib/wp/sync-handler/cache-index.js

```diff
@@ -65,12 +65,16 @@ export const cacheIndex = {
 	 * @param {String} key - record key
 	 * @return {Promise} promise
 	 */
-	addItem( key ) {
+	addItem( key, pageSeriesKey = null ) {
```

Comment:
It's only added for records that need it (i.e., those that are part of a page-series), and yes it's specific to lists that support pagination with a `next_page` attribute in their responses. We need a key in order to identify all records that are in the same page-series.

> Also can you update @jsdoc here?

for sure.